### ISSUE TITLE
Fix bogus warnings

### ISSUE THREAD

**Issue by narensankar0529** [RhetTbull/osxphotos#365]
Currently osxphotos will output a warning - Some keywords exceed max IPTC Keyword length of 64 - when keywords are longer than can be fit in IPTC keyword field. However this will be output even if not using the IPTC field - since keywords are currently saved in the XMP dc:Subject field which does not have this limitation

**Comment by RhetTbull** [comment]
The same function that computes the keywords (e.g. if using a keyword template) is used for both XMP and IPTC.  I could move length check closer to the exiftool code.

**Comment by narensankar0529** [comment]
That would. be great. The warning is not an issue in that the XMP is created correctly. It just adds a lot of clutter to the output which I am logging to detect errors and so just worried that a real error might slip through in the flood of all all warning. 

I am using --keyword-template "{place.name.area_of_interest}" and since this is where the warning gets generated

**Comment by RhetTbull** [comment]
So exiftool will issue a warning and truncate the keyword if it's too long for IPTC:Keywords so I might just remove the check from osxphotos and let exiftool give the warning (osxphotos will log/report any warnings from exiftool):

```sh
exiftool -IPTC:Keywords='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx' 2020/12/IMG_3092_edited.jpeg
Warning: [Minor] IPTC:Keywords exceeds length limit (truncated) - 2020/12/IMG_3092_edited.jpeg
    1 image files updated
```

**Comment by narensankar0529** [comment]
Yeah. That will work. Just use the warming from exiftool if the IPTC is being written

**Comment by RhetTbull** [comment]
This is fixed in [v0.48.9](https://github.com/RhetTbull/osxphotos/releases/tag/v0.40.9)
